client/cl_utils.lua

```diff
@@ -7,3 +7,14 @@ function comma_value(amount)
 
     return amount
 end
+
+---@return table
+function FindPlayers()
+	local playerCoords = GetEntityCoords(cache.ped)
+    local closePlayers = {}
+    for _, v in pairs(GetPlayersFromCoords(playerCoords, 10)) do
```

Comment:
On second thought, if you use https://overextended.dev/ox_lib/Modules/GetNearbyPlayers/Client

Then you can filter out the calling player, since you don't want it in the first place.

Reply:
I'll give this a go and see if it eliminates the above call to GetPlayerServerId

Reply:
You'll still need to use GetPlayerServerId, but you can get rid of your filtering if condition to filter out the player.

Reply:
This did the trick. Still needed the conversion to Server Ids but I got a tighter loop for it.